README.md

```diff
@@ -129,6 +129,41 @@ You can pull your queries directly from a standard GitHub search! If you want to
 
+## Release Process
+
+Good First Issue follows a relatively strict release process intended to ensure the spice flows.
+
+### Versioning
+
+| Semantic Version | Type | Reason |
+|-------------------|-------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| Major (**x**.x.x) | Breaking changes and non-trivial upgrades | Ensuring that end-users can rely on Good First Issue not breaking however they're consuming it |
+| Minor (x.**x**.x) | Project additions, other feature additions | Following the SemVer standard, project additions and feature additions are backwards-compatible enhancements. We generally try to ship one addition per Minor. |
+| Patch (x.x.**x**) | Bug fixes, minor enhancements to metadata and content | Tiny, hardly visible fixes to improve UX/DX or fix the module |
+
+## Labels and Milestones
+We use both GitHub Labels and Milestones to track releases. Since project additons count as a minor release, we prefer to space those out and ship them individually rather than shipping many at once. This pace may be revised later, but for now it introduces the need for a release queue and setting things up to be released ahead of them acutally being released.
+
+We use the release queue [label](https://github.com/bnb/good-first-issue/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) and [milestone](https://github.com/bnb/good-first-issue/milestone/16) to queue up PRs that have been reviewed and are ready to be released.
+
+Once a PR is ready to be released, a milestone will be added that correlates to the SemVer version it will be released in. Ideally this will _eventually_ be used for changelog tracking but for now it's just a good way to keep organized. To keep things tidy, once a new version has shipped the milestone will be closed out.
+
+### Local testing
+Prior to each release, whoever is releasing should be testing the release locally to ensure that the code is working as expected. This would include either running `npm i -g` or `npm link` in the PR branch and then testing whatever the PR is adding. Ensuring the experience isn't broken is vital.
+
+It is worth noting that we limit the file we publish to npm with the `files` property in [`package.json`](https://github.com/bnb/good-first-issue/blob/master/package.json#L15). This property prevents code that's not explicitly listed from being shipped. We have had a situation where local testing and the published module differed because a PR was merged that added needed code in a directory that wasn't included. So, what works on your machine may not work for the end user.
```

Comment:
Can we avoid linking inside the file? It might not work when we modify `package.json`.

Reply:
Yes the _line reference_ would fail after some changes

Reply:
neat 👍